projects/front-end-library/src/lib/scss/dependencies/bifrost/_border.scss

```
@@ -13,8 +13,8 @@
     &-color-tertiary {
         border-color: var(--bf-color-stroke-tertiary) !important;
     }
-
-    &-color-highlight {
-        border-color: var(--bf-color-higlight) !important;
+    // #ffffff | #050504
+    &-highlight{
+    border-color: var(--bf-color-stroke-highlight) ;
```

```suggestion
    &-highlight {
      border-color: var(--bf-color-stroke-highlight);
```

Alignement et espacement